ext/build_defaults.yaml

```diff
@@ -21,6 +21,9 @@ build_msi:
   facter:
     ref: 'refs/tags/2.3.0'
     repo: 'git://github.com/puppetlabs/facter.git'
+  cfacter:
+    ref: 'cfacter-553c5dd5eaeda6ee6c8591ab536e776390c58f8b.zip'
+    repo: 'http://builds.puppetlabs.lan/cfacter/553c5dd5eaeda6ee6c8591ab536e776390c58f8b/artifacts/windows/cfacter-553c5dd5eaeda6ee6c8591ab536e776390c58f8b.zip'
```

Comment:
repo is a pretty loosely defined term here. Have we considered naming these fields differently for binaries?